extra_data/components.py

```diff
@@ -796,9 +808,7 @@ def ndarray(self, *, fill_value=None, out=None, roi=(), astype=None, module_gaps
 
         module_dim = self.det.n_modules if module_gaps else len(self.modno_to_keydata)
```

Comment:
This does not seem to be required anymore.

Reply:
Thanks, good catch.